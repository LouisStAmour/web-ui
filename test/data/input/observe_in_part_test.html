<!doctype html>
<!--
Copyright (c) 2013, the Dart project authors.  Please see the AUTHORS file
for details. All rights reserved. Use of this source code is governed by a
BSD-style license that can be found in the LICENSE file.
-->
<html lang="en">
<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
  <script type="application/javascript" src="testing.js"></script>
</head>
<body>
  <span>{{message}}</span>
  <script type="application/dart">
    library observe_in_part_test;
    import 'dart:async';
    import 'dart:html';
    part 'observe_in_part.dart';
    main() {
      Timer.run(() {
        message = '$message, this test passed!';
        window.postMessage('done', '*');
      });
    }
  </script>
</body>
</html>
