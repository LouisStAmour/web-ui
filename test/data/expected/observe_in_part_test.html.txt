Content-Type: text/plain
layer at (0,0) size 800x600
  RenderView at (0,0) size 800x600
layer at (0,0) size 800x36
  RenderBlock {HTML} at (0,0) size 800x36
    RenderBody {BODY} at (8,8) size 784x20
      RenderInline {SPAN} at (0,0) size 132x19
        RenderText {#text} at (0,0) size 132x19
          text run at (0,0) width 132: "hello, this test passed!"
      RenderText {#text} at (0,0) size 0x0
      RenderText {#text} at (0,0) size 0x0
